README.md

```diff
@@ -1,21 +1,83 @@
 Terraform Open Telekom Cloud Provider
 =====================================
+[Documentation](https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs) | [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
```

Comment:
Maybe wrap `documentation` reference in a badge? E.g. ![documentation](https://img.shields.io/badge/doc-registry-blue) or ![documentation](https://img.shields.io/badge/documentation-blue)